### ISSUE TITLE
Duplicate type names for aliased types

### ISSUE THREAD

**Issue by robbertkl** [Urigo/graphql-scalars#125]
Perhaps this issue is related to me using graphql-modules, but I'm getting errors for each of the aliased types (`UnsignedInt`, `UnsignedFloat`, `Long`) when importing all the types into my `GraphQLModule`:

```
There can be only one type named "BigInt".
There can be only one type named "NonNegativeFloat".
There can be only one type named "NonNegativeInt".
```

They're exported as aliases here:

https://github.com/Urigo/graphql-scalars/blob/9ae9e8932f267028ceea52f71dd51c1ccc607799/src/resolvers/index.ts#L34

But they still have an explicitly set `name` property here:

https://github.com/Urigo/graphql-scalars/blob/9ae9e8932f267028ceea52f71dd51c1ccc607799/src/resolvers/NonNegativeInt.ts#L8

Obviously, I can work around this by only importing the types I need, but I prefer to import everything at once. 

Any idea why this is not working like in the README? If it's only broken when using graphql-modules, why does it work in "plain Apollo mode"? Is the name property not used in that case?

**Comment by ardatan** [comment]
@robbertkl Could you share a reproduction repo to see the exact scenerio?

**Comment by robbertkl** [comment]
Minimal reproduction code isn't that large, so I'll just put it here:

```js
const { ApolloServer, gql } = require('apollo-server');
const { typeDefs, resolvers } = require('graphql-scalars');
const { GraphQLModule } = require('@graphql-modules/core');

const ScalarsModule = new GraphQLModule({
  typeDefs,
  resolvers,
});

const AppModule = new GraphQLModule({
  imports: [ScalarsModule],
  typeDefs: gql`
    type Query {
      test: Int
    }
  `,
})

new ApolloServer({ 
  schema: AppModule.schema,
 });
```

The reason I'm using ApolloServer `schema` instead of `modules` is: https://github.com/Urigo/graphql-modules/issues/628

**Comment by ardatan** [comment]
@robbertkl Could you test it with the canary version?
`1.0.3-8bbccd5.0`

**Comment by ardatan** [comment]
Available in v1.0.3!

**Comment by robbertkl** [comment]
Sorry, did not have a chance to test it yet. Great it's already released. Thanks @ardatan !
